src/targets/slack.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Bristol
+ * Copyright 2014-2016 Tom Shawver
+ */
+
+'use strict'
+
+const request = require('request-promise')
+
+/**
+ * The Slack target posts to a Slack channel via POST to a webhook.
+ * @param {Object} options Map of options for Slack
+ * @param {string} options.webhook Slack webhook URL
+ * @param {string} options.icon_emoji Slack emoji code for message icon, ex: :boom:
+ * @param {string} options.username Username to display
+ * @param {string} options.channel The channel to send to in Slack
+ * @param {string} severity Unused
+ * @param {Date} date Unused
+ * @param {string} message The message to be sent to Slack
+ */
+module.exports = (options, severity, date, message) => {
+  let text
+  if (message.indexOf('\n') >= 0) {
```

Comment:
Two thoughts:

- Since any formatter could be used, it might make sense to make the newline splitting an option. Even defaulting it to false, so we don't do anything unexpected unless the user is explicitly requesting it.

- It might make sense to do `> 0` rather than `>= 0`, because if it's the latter, you don't have any text to make bold anyway.